packages/blade/src/components/SideNav/_decisions/decisions.md

```diff
@@ -0,0 +1,418 @@
+| --------- | ----------------------------------------------------- | -------- | ----------------- |
+| title     | Only applicable in L2. Becomes the title of the Level | string   |                   |
+| children  | children slot. Accepts SideNavLink as children        | JSX      |                   |
+
+<table>
+<tr>
+<td>
+
+```jsx
+<SideNav>
+  {/* L1 Items */}
+  <SideNavLink title="L1 Item" />
+
+  <SideNavLink title="L2 Trigger">
+    {/* L2 Level */}
+    <SideNavLevel title="L2 Menu Heading">
+      <SideNavLink />
+      <SideNavLink />
+      <SideNavLink title="L3 Trigger">
+        {/* L3 Level */}
+        <SideNavLevel>
+          <SideNavLink />
+          <SideNavLink />
+        </SideNavLevel>
+      </SideNavLink>
+    </SideNavLevel>
+</SideNav>
+```
+
+</td>
+
+<td>
+<img width="500px" src="navlevels-eg.png" />
+</td>
+</tr>
+</table>
+
+### SideNavSection
+
+| **Props**       | **Description**                                              | **Type** | **Default Value** |
+| --------------- | ------------------------------------------------------------ | -------- | ----------------- |
+| title           | title of the section                                         | string   |                   |
+| maxVisibleItems | Number of items visible (rest go inside +x more collapsible) | number   | undefined         |
+| children        | Children slot. For SideNavLink children items                | JSX      |                   |
+
+<table>
+<tr>
+<td>
+
+<!-- prettier-ignore -->
+```jsx
+<SideNav>
+  <SideNavSection 
+    title="OFFERINGS SECTION" 
+    maxVisibleItems={3}
+  >
+    {/* All SideNavLink items */}
+  </SideNavSection>
+</SideNav>
+```
+
+</td>
+
+<td>
+<img width="200px" src="navsection-eg.png" />
+</td>
+</tr>
+</table>
+
+### SideNavFooter
+
+| **Props** | **Description** | **Type** | **Default Value** |
+| --------- | --------------- | -------- | ----------------- |
+| children  | Children slot   | JSX      |                   |
+
+<table>
+<tr>
+<td>
+
+<!-- prettier-ignore -->
+```jsx
+<SideNav>
+  {/* ... Other Items */}
+  <SideNavFooter>
+    <Box 
+      display="flex" 
+      paddingY="spacing.4" 
+      paddingX="spacing.3" 
+      justifyContent="spacing-between"
+    >
+      <Box display="flex" gap="spacing.3">
+        <Indicator color="positive" />
+        <Text>Test Mode</Text>
+      </Box>
+      <Switch />
+    </Box>
+    <SideNavLink 
+      icon={GearIcon}
+      href="/settings" 
+      title="Settings" 
+    >
+      {/* L2 SideNavLink items */}
+    </SideNavLink>
+  </SideNavFooter>
+</SideNav>
+```
+
+</td>
+
+<td>
+<img width="300px" src="navfooter-eg.png" />
+</td>
+</tr>
+</table>
+
+## Accessibility
+
+1. All items should be accessible by `TAB`. Including going between levels L1, L2, L3
+2. We should use Blade's `SkipNav` utility to provide option of skipping nav and going to content
```

Comment:
what is the SkipNav utility can you share more context about it?

Reply:
We have this utility in blade - https://blade.razorpay.com/?path=/docs/components-accessibility-skipnav--docs

So if you start tabbing, you see a Skip to content link which you can click to skip navigation and go to content directly